lib/module.js

```diff
@@ -447,6 +447,10 @@ Module._load = function(request, parent, isMain) {
   }
 
   var module = new Module(filename, parent);
+  if (opts !== undefined && opts.dlopenFlags) {
+    assert(util.isNumber(opts.dlopenFlags), 'dlopenFlags must be a number');
```

Comment:
This should not be an assert since this is checking a user provided value.
Instead, you'll want something like:

```
if (!util.isNumber(opts.dlopenFlags)) {
  const errors = require('internal/errors');
  throw new errors.TypeError('ERR_INVALID_OPT_VALUE',
                             'dlopenFlags', opts.dlopenFlags);
}
```